testify/test_rerunner.py

```diff
@@ -32,23 +42,29 @@ def discover(self):
         else:
             tests = open(self.filename)
 
-        with tests as tests:
```

Comment:
now this file doesn't get closed in a timely manner -- is this the `RuntimeWarning` that you silenced? (seems incorrect to me)